scripts/ticketmaster-API.py

```diff
@@ -17,63 +18,80 @@
-
-        main_venue = event['_embedded']['venues'][0]
-        main_venue_name = main_venue['name']
-
-        main_venue_location = main_venue.get('location')
-        main_venue_lat = main_venue['location'][
-            'latitude'] if main_venue_location else None
-        main_venue_lon = main_venue['location'][
-            'longitude'] if main_venue_location else None
-
-        artists = event['_embedded'].get('attractions')
-        main_artist_id = artists[0]['id'] if artists else ''
-        main_artist_name = artists[0]['name'] if artists else ''
-        main_artist_genre = artists[0]['classifications'][
-            0]['genre']['name'] if artists else ''
-
-        query = """INSERT INTO ticketmaster_events
-                    (ticketmaster_id, local_date, event_genre,
-                    event_subgenre, venue, venue_lat, venue_long,
-                    artist_id, artist_name, artist_genre) VALUES
-                    (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)"""
-        values = (event_id, local_date, main_genre_name, sub_genre_name,
-                  main_venue_name, main_venue_lat, main_venue_lon,
-                  main_artist_id, main_artist_name, main_artist_genre)
-
-        cursor.execute(query, values)
-        db.commit()
-
-    time.sleep(1)
+
+    for date in date_list:
+        time.sleep(.21)
+
+        params = {'stateCode': us_state,
+                  'classificationName': classificationName,
+                  'apikey': apikey,
+                  'startDateTime': date[0],
+                  'endDateTime': date[1],
+                  'size': size
+                  }
+        r = requests.get(url=URL, params=params)
+        data = r.json()
+
+        total_events = data['page']['totalElements']
+
+        if total_events > 0 :
+            events = data['_embedded']['events']
+
+            for index, event in enumerate(events):
+                local_date = event['dates']['start']['localDate']
+                event_id = event['id']
+
+                main_genre = event['classifications'][0].get('genre')
+                main_genre_name = main_genre['name'] if main_genre else ''
+                sub_genre = event['classifications'][0].get('subGenre')
+                sub_genre_name = sub_genre['name'] if sub_genre else ''
+
+                main_venue = event['_embedded']['venues'][0]
+                main_venue_name = main_venue['name']
+
+                main_venue_location = main_venue.get('location')
+                main_venue_lat = main_venue['location'][
+                    'latitude'] if main_venue_location else None
+                main_venue_lon = main_venue['location'][
+                    'longitude'] if main_venue_location else None
+
+                artists = event['_embedded'].get('attractions')
+                main_artist_id = artists[0]['id'] if artists else ''
+                main_artist_name = artists[0]['name'] if (artists is not None and
```

Comment:
I think you can just say:
```
main_artist_name = artists[0]['name'] if (artists and artists[0]['name']) else ''
```
To make the line a bit cleaner. 
(If you can't, leave it as is)

Reply:
(Same for genre)

Reply:
Was able to simplify the `artists` portion, but had to leave the `'name' in in artists[0]` portion. Same for genre